docs/crd.md

```diff
@@ -110,10 +110,12 @@ spec:
     autoNatOutgoing: false                            # Optional, Overlay Network only. 
                                                       # If pods in this sunbet can access to addresses outside 
                                                       # the cluster by NAT.
+                                                      # If it's empty, it works the same as being set true.
```

Comment:
nit: simplify this as `default is true` ?

Reply:
done~